src/dotnet/commands/dotnet-compile/Program.cs

```diff
@@ -71,16 +71,35 @@ private static bool OnExecute(List<ProjectContext> contexts, CompilerCommandApp
 
             var managedOutput = outputPaths.CompilationFiles.Assembly;
 
+            // Create the library exporter
+            var exporter = context.CreateExporter(args.ConfigValue);
+
+            // Gather exports for the project
+            var dependencies = exporter.GetDependencies().ToList();
+
+            var references = new List<string>();
+            foreach (var dependency in dependencies)
```

Comment:
Can you please move all the code get runtime references to a separate function so I can re-use it?